app.js

```diff
@@ -66,10 +66,10 @@ app.use(function(req, res, next) {
 });
 
 app.get('/validate', function (req, res, next) {
-  if (isValidToken(req.query.token)) {
-    sessionExists(req.query.token).then(function() {
+  var apiToken = req.query.token;
+  if (isValidToken(apiToken)) {
```

Comment:
Not sure I understand what you mean by `for use`; It does return `true` if the token is valid, `false` if it isn't.